lib/bigtable/family.js

```diff
@@ -0,0 +1,311 @@
+/*!
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ /*!
+ * @module bigtable/family
+ */
+
+'use strict';
+
+var createErrorClass = require('create-error-class');
+var nodeutil = require('util');
+var is = require('is');
+
+/**
+ * @private
+ */
+var FamilyError = createErrorClass('FamilyError', function(name) {
+  this.message = 'Column family was not found in table: ' + name;
+  this.code = 404;
+});
+
+/**
+ * @type {module:common/grpcServiceObject}
+ * @private
+ */
+var GrpcServiceObject = require('../common/grpc-service-object.js');
+
+/**
+ * Create a Family object to interact with your table column families.
+ *
+ * @constructor
+ * @alias module:bigtable/family
+ *
+ * @example
+ * var gcloud = require('gcloud');
+ *
+ * var bigtable = gcloud.bigtable({
+ *   keyFilename: '/path/to/keyfile.json',
```

Comment:
That's currently how we authenticate all requests IIRC (@stephenplusplus). We use a module called `google-auto-auth` that sits on top of `google-auth-library-nodejs` to create the auth client for us.